### DISCUSSION TITLE
Where is visual editor?

### DISCUSSION THREAD

**Discussion by georgyfarniev** [community/community#26300] [Actions]
<p>Hello. I can see visual editor on github actions landing and in some example, but I cannot find it. is it removed or temporarily disabled? If not, how can I find visual editor? Thanks.</p>

**Comment by oldskool** [comment, marked as answer]
<p>The GitHub web editor supports workflow files, but it has to be an existing file for it to be recognized. Creating a new file through the web interface doesn’t work (yet). So simply open the workflow file you want to edit, click the pencil icon in the top right of that file and the editor will show up with syntax highlighting, autocomplete and such.</p>

**Reply by agaudreault** [reply]
You can take a look at [actionforge.dev](https://www.actionforge.dev/) for a visual editor of GitHub Actions in VSCode.

**Comment by st-schneider** [comment]
<p>So that currently is not the case anymore or is it? with the workflow/.yml files theres seem to be no visual workflow editor anymore.</p>

**Comment by georgyfarniev** [comment]
<p>Please correct me if I’m wrong, as I understand, *.workflow files are old syntax, and *.yml is new, right? So is there plan to add visual editor to new syntax? It looks really great on pictures and actions landing itself, so would be great to have it.</p>

**Comment by wisehackermonkey** [comment]
<p>writing in 2020:</p>
<p>Im having the same issue, cant see the visual editor…:frowning:</p>

**Comment by danielo515** [comment]
<p>Same problem here. The documentation is messy and not clear about this. You see the visual editor all over the place on the actions page, but when it is time to try it you don’t find any reference about how or where.</p>

**Comment by sblantipodi** [comment]
<p>It doesn’t work anymore.<br>
No visual editor for me, no matter what I do.</p>

**Comment by dospunk** [comment]
<p>Also can’t see the visual editor at all. Really disappointing, since it seems like a great tool.</p>

**Comment by francisfuzz** [comment]
<p>:wave: Hey everyone!  I’m aware of the  <a href="https://github.blog/2019-10-01-new-workflow-editor-for-github-actions/">workflow editor</a> as well as the recently shipped <a href="https://github.blog/changelog/2020-12-08-github-actions-workflow-visualization/">workflow visualization</a>.</p>
<p>Curious to get more information around the visual editor that’s being referenced here–is it the one that’s pictured in <a href="https://github.com/features/actions">our GitHub Actions marketing page</a>, or something else? :thought_balloon:</p>
<hr>
<p><img src="https://github.githubassets.com/images/modules/site/features/actions-workflow.svg" alt="visual-editor-in-marketing-page" width="690" height="284"></p>
<hr>

**Comment by yeongrokgim** [comment]
<p>For anyone who is looking for interactive visual editor for workflow file:<br>
this might be helpful: <a href="https://github.com/cschleiden/github-actions-hero" class="inline-onebox" rel="noopener nofollow ugc">GitHub - cschleiden/github-actions-hero: 🚧 Interactive tutorial and visualizer for GitHub Actions workflows 🚧</a></p>

**Comment by agaudreault** [comment]
You can take a look at https://www.actionforge.dev/ for a visual editor of GitHub Actions in VSCode.
